updater/lib/dependabot/updater/operations/group_update_all_versions.rb

```diff
@@ -99,8 +99,9 @@ def perform
         sig { returns(Dependabot::Updater::ErrorHandler) }
         attr_reader :error_handler
 
+        # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity
```

Comment:
I think these are generally good cops to pay attention to, but I suspect the violations are due to the feature flagging. As such, I think it makes sense to temporarily ignore them.